company-profiles/zootools.md

```diff
@@ -0,0 +1,44 @@
+# Zootools
```

Comment:
This may need to match the filename exactly (zootools instead of Zootools)

Reply:
> This may need to match the filename exactly (zootools instead of Zootools)

Thanks , I was really confused why the deployment failed.